rule.go

```diff
@@ -914,3 +911,14 @@ func (r *Rule) HasVar(s string) bool {
 	}
 	return false
 }
+
+//getSidMgs returns a string representing a sidmsg.map entry
+func (r *Rule) getSidMgs(s string) string {	
```

Comment:
This seems like a function that should be exported so it's usable in outside packages. Additionally abbreviated "message" is backward. The `s string` parameter should be unneeded here.

Function signature should look like
`func (r *Rule) GetSidMsg() string`

Reply:
done.